src/components/product-switcher/style.module.css

```diff
@@ -0,0 +1,104 @@
+.productSwitcher {
+  position: relative;
+}
+
+.switcherButton {
+  align-items: center;
+  background-color: transparent;
+  border-radius: 4px;
+  border: 1px solid var(--gray-foreground-tertiary);
+  color: white;
+  display: flex;
+  justify-content: space-between;
+  padding: 8px;
+  position: relative;
+  width: 220px;
+
+  &:hover {
+    cursor: pointer;
+  }
+
+  /* TODO - create variables for these colors */
+  &:focus {
+    outline: none;
+    background: #06142d;
+    border: 1px solid var(--blue-vibrant);
+    box-shadow: 0 0 0 3px #0c295a;
+    border-radius: 4px;
+  }
+}
+
+.switcherCaret {
+  color: white;
+  margin-right: 4px;
+}
+
+/*
+FIXME: box-shadow value is copied from Figma, but it's not rendering in the
+browser the same as it does in Figma.
+*/
+.switcherOptionList {
+  background-color: white;
+  border-radius: 5px;
```

Comment:
Design observation...the button radius is 4px, while this is 5px...I wonder if we could align on 4 for this element 💭

Reply:
Good catch, I'll ask about this and double check that this is what we want!

Reply:
I asked about this and we do indeed want consistent values! The value is supposed to be `5px` and I've made the updates.